src/libraries/System.Drawing.Common/src/System/Drawing/macFunctions.cs

```diff
@@ -211,13 +211,12 @@ internal static void ReleaseContext(IntPtr port, IntPtr context)
         #region Cocoa Methods
         [DllImport("libobjc.dylib")]
         public static extern IntPtr objc_getClass(string className);
-        [DllImport("libobjc.dylib")]
-        public static extern IntPtr objc_msgSend(IntPtr basePtr, IntPtr selector, string argument);
-        [DllImport("libobjc.dylib")]
-        public static extern IntPtr objc_msgSend(IntPtr basePtr, IntPtr selector);
-        [DllImport("libobjc.dylib")]
-        public static extern void objc_msgSend_stret(ref Rect arect, IntPtr basePtr, IntPtr selector);
         [DllImport("libobjc.dylib", EntryPoint = "objc_msgSend")]
+        public static extern IntPtr intptr_objc_msgSend(IntPtr basePtr, IntPtr selector);
+        [DllImport("libobjc.dylib", EntryPoint = "objc_msgSend_stret")]
+        public static extern void void_objc_msgSend_stret_rrect(ref Rect arect, IntPtr basePtr, IntPtr selector);
```

Comment:
Would this look better as `rect_objc_msgSend_stret(out Rect arect` ? The Xamarin examples are 100% inconsistent - majority of them are using `out`, a few exception are using `ref` for some reason.

Reply:
I don't think it particularly matters. We could also have the Rect type as the return value if we want.